src/LongRange/LRHandlerTemp.h

```diff
@@ -127,6 +131,7 @@ class LRHandlerTemp : public LRHandlerBase
   inline mRealType evaluateLR(mRealType r)
   {
     mRealType v = 0.0;
+    if (r>=LR_rc) return 1./r;
```

Comment:
This should be return myfunc(r,rinv).  1/r assumes coulomb and breaks the generic handling of myFunc.

Reply:
Great catch! I just pushed an update